summarize/src/main.rs

```diff
@@ -75,6 +85,19 @@ fn write_results_json(
     Ok(())
 }
 
+fn aggregate(opt: AggregateOpt) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let profiles = opt
+        .files
+        .into_iter()
+        .map(|file| ProfilingData::new(&file))
+        .collect::<Result<Vec<_>, _>>()?;
+
+    // FIXME(eddyb) return some kind of serializable data structure from `aggregate_profiles`.
```

Comment:
This would be nice to have since it would probably be part of any plans to implement the aggregation feature for perf.rlo.